.circleci/config.yml

```diff
@@ -255,6 +255,10 @@ jobs:
           name: Compile HAL for all STM32F7
           command: |
             (cd test/all && python3 run_all.py stm32f7)
+      - run:
+          name: Examples STM32F7 Series
+          command: |
+            (cd examples && ../tools/scripts/examples_compile.py nucleo_f746zg)
```

Comment:
No? The CI job you're looking for is around line 100.

Reply:
And the example was already added in 10187f7

Reply:
Sorry. My mistake.